src/VisualStudio/Xaml/Impl/Features/AutoInsert/XamlAutoInsertResult.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.VisualStudio.LanguageServices.Xaml.Features.AutoInsert
+{
+    internal class XamlAutoInsertResult
+    {
+        public TextChange TextChange { get; set; }
```

Comment:
I think in general our preference for these would be to make them readonly with a ctor, but I'll leave it up to you.

Reply:
I saw quite a few of those used by LSP handlers so I kept it simple. It will make adding additional params easier.